src/Containers/Pages/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './LandingPage.css';
+import Helmet from 'react-helmet';
+
+const LandingPage = () => {
+    return (
+        <>
+            <Helmet bodyAttributes={{style: 'background-color : #FFD275'}}/>
```

Comment:
I think using Helmet here was a bit of an overkill, Instead of changing the <body> directly in a page, try surrounding it with <div> and add styles to the <div>